textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/DataConstants.java

```diff
@@ -11,4 +11,8 @@
     public static final String INDEX_DIR = "../index";
     public static final String SCAN_QUERY = "*:*";
     public static final int MAX_RESULTS = 100;
+
+    public static enum SourceOperatorType {
+        SCANOPERATOR, KEYWORDOPERATOR, PHRASEOPERATOR

```

Comment:
Just want to confirm: currently we treat `PHRASEOPERATOR` NOT as a special kind of `KEYWORDOPERATOR`.